### ISSUE TITLE
New scroll-into-view modifier

### ISSUE THREAD

**Issue by ygongdev** [elwayman02/ember-scroll-modifiers#261]
**Description**
Adding a new scroll-into-view functional modifier that mimics the native [scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView).

**API**
The arguments are chosen to closely mimc the API of the native scrollIntoView, which accepts an object parameter. We also introduce a boolean/promise to conditionally trigger the scrollIntoView functionality.

**Examples**
```hbs
<div {{scroll-into-view options=this.scrollOptions shouldScroll=this.shouldScroll}}></div>
```

**Comment by miguelcobain** [comment]
Was this addressed by #360?

**Comment by elwayman02** [comment]
Yup, thanks!
